v23.1/fips-compliance.md

```diff
@@ -0,0 +1,355 @@
+
+The Cryptographic Module Validation Program (CMVP) validates cryptographic modules to FIPS 140-2 and other cryptography-based standards. When a cryptographic module or library has a FIPS 140-2 certificate, it has been tested and formally validated under the CMVP as meeting the requirements for FIPS 140-2.
+
+FIPS-ready CockroachDB binaries and Docker images are available for CockroachDB v23.1.0 and above. CockroachDB FIPS-ready runtimes run only on Intel 64-bit Linux systems.
+
+FIPS-ready binaries are built using [Red Hat's Go FIPS with OpenSSL toolchain](https://github.com/golang-fips/go), which contains the necessary modifications for the Go crypto library to use an external cryptographic library in a manner compatible with FIPS 140-2. FIPS-ready CockroachDB delegates cryptographic operations to the host operating system's OpenSSL libraries and commands, rather than Go's cryptographic libraries. When the installed OpenSSL has a FIPS 140-2 certificate and [FIPS mode is enabled in the Linux kernel](#enable-fips-mode-in-the-linux-kernel), the CockroachDB runtime is suitable for workloads that are subject to FIPS 140-2 requirements.
+
+For details about cryptographic algorithms and key lengths used by CockroachDB. refer to [Details About Cryptographic Algorithms](#details-about-cryptographic-algorithms).
+
+### FIPS-ready features
+
+When you use a FIPS-ready CockroachDB runtime and each cluster node's OpenSSL is FIPS-validated and configured correctly, Cockroach Labs has tested the following features to ensure that their cryptographic operations meet the requirements of FIPS 140-2:
+
+- [Encryption At Rest](security-reference/encryption.html#encryption-keys-used-by-cockroachdb-self-hosted-clusters)
+- [Encrypted Backups](take-and-restore-encrypted-backups.html)
+- [Change Data Capture to Kafka over TLS](create-changefeed.html#query-parameters)
+- [Certificate-based Node-to-Node and Client-to-Node Authentication](authentication.html)
+- [SASL SCRAM-SHA-256 Password Authentication](security-reference/scram-authentication.html)
+- [Built-in SQL cryptographic functions](functions-and-operators.html#cryptographic-functions)
+
+  {{site.data.alerts.callout_danger}}
+  In a FIPS-ready runtime, Cockroach Labs recommends that you avoid using cryptographic operations that are not supported by FIPS 140-2. For example, generating an MD5 hash is not compatible with FIPS 140-2, because MD5 is not a FIPS-validated algorithm. Use algorithms and functions that do not comply with the standard at your own risk.
```

Comment:
```suggestion
  When running a FIPS-ready runtime, Cockroach Labs recommends that you avoid using cryptographic operations that are not supported by FIPS 140-2. For example, generating an MD5 hash is not compatible with FIPS 140-2, because MD5 is not a FIPS-validated algorithm. Use algorithms and functions that do not comply with the standard at your own risk.
```